### PR TITLE
chore: update kolibri urls

### PR DESCRIPTION
**Pull request by deleonio**
We have a new documentation site with seo urls.

### CODE REVIEW

**Comment by deleonio** [comment]
Please rerun the IPR-Check. Thank you.

File: site/src/pages/components/table.mdx [hunk 1]
```diff
@@ -124,6 +124,7 @@ Data grid implements the [grid role](https://www.w3.org/TR/core-aam-1.1/#role-ma
...
-| Evergreen   | [Table](https://evergreen.segment.com/components/table)                                                                                               | [Playground](https://codesandbox.io/s/evergreen-playground-component-fjbly)                          | `<div>`                       | -                                                                |
+| Evergreen   | [Table](https://evergreen.segment.com/components/table)                                                                                               | -                                                                                                    | `<table>`                     | -                                                                |
+| KoliBri     | [Table](https://public-ui.github.io/docs/components/table)                                                                                            | [Playground](https://codesandbox.io/s/evergreen-playground-component-fjbly)                          | `<div>`                       | -                                                                |
```
Comment (andrico1234) [inline site/src/pages/components/table.mdx:128]: it looks like you KoliBri is sandwiched between the Evergreen table link and playground

Comment (andrico1234) [inline site/src/pages/components/table.mdx:128]:
```suggestion
| KoliBri     | [Table](https://public-ui.github.io/docs/components/table)                                                                                            | -                     | `<div>`                       | -                                                                |
```

**Review by andrico1234 — CHANGES_REQUESTED** [review]
Thanks for making the change, I have just one requested change and then I'm happy for this to be merged in

**Comment by deleonio** [comment]
@andrico1234 really cool, thank you.

**Review by andrico1234 — APPROVED** [review]

**Comment by andrico1234** [comment]
Thanks for this! :)

**Pull request merged**